src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyInputException;
+import duke.exceptions.IllegalInputException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LINE = "____________________________________________________________\n";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!\n";
+    public static boolean canExit = false;
+
+    /**
+     * Prints message after adding To-Do/Deadline/Event duke.task
+     *
+     * @param myList
+     */
+    public static void printTaskMessage(ArrayList<Task> myList) {
+        System.out.println(LINE
+                + "Got it. I've added this duke.task:\n"
+                + myList.get(myList.size() - 1) + "\n"
```

Comment:
Try to avoid magic literals. There has been a few more instances in the rest of your code. You can use a named constant makes the code easier to understand because the name tells us more about the meaning of the number.